include/mbedtls/pk.h

```diff
@@ -361,24 +361,35 @@ int mbedtls_pk_setup(mbedtls_pk_context *ctx, const mbedtls_pk_info_t *info);
 /**
  * \brief           Initialize a PK context to wrap a PSA key.
  *
- * \note            This function replaces mbedtls_pk_setup() for contexts
- *                  that wrap a (possibly opaque) PSA key instead of
- *                  storing and manipulating the key material directly.
+ *                  This function helps creating a PK context which wraps a
+ *                  PSA key. The PSA wrapped key must:
+ *                  * remain valid as long as the wrapping PK context is in use,
+ *                    that is at least between the point this function is
+ *                    called and the point mbedtls_pk_free() is called on this
+ *                    context;
+ *                  * be a key pair;
+ *                  * be an EC or RSA type (DH is not suported in PK module).
+ *
+ *                  Under the hood PSA functions are used to perform the required
+ *                  operations and, based on the key type, used algorithms will be:
+ *                  * EC:
+ *                      * verify, verify_ext: #PSA_ALG_ECDSA_ANY;
+ *                      * sign, sign_ext: try #PSA_ALG_DETERMINISTIC_ECDSA()
+ *                        first and, in case it fails, try with #PSA_ALG_ECDSA().
+ *                  * RSA:
+ *                      * sign, sign_ext: use the algorithm associated with the
```

Comment:
“algorithm associated with” is not well-defined. More precisely, based on it uses the primary algorithm in the key's policy.